nvflare/app_opt/xgboost/histogram_based_v2/adaptors/grpc_client_adaptor.py

```diff
@@ -99,6 +102,12 @@ def start(self, fl_ctx: FLContext):
         if not self.num_rounds:
             raise RuntimeError("cannot start - num_rounds is not set")
 
+        if not self.training_mode:
+            raise RuntimeError("cannot start - training_mode is not set")
+
+        if not self.xgb_params:
+            raise RuntimeError("cannot start - xgb_params is not set")
+
```

Comment:
these checks should be moved into base "xgb_adaptor.py"?

Reply:
Those checks are redundant, already done in xgb_adaptor.py. Removed.